### PR TITLE
Encode only string types

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: c2cgeoportal/scaffolds/__init__.py [hunk 1]
```diff
@@ -162,6 +163,7 @@ def pre(self, command, output_dir, vars):
             project = load(file("project.yaml", "r"))
             if "template_vars" in project:
                 for key, value in project["template_vars"].items():
-                    vars[key] = value.encode("utf-8")
+                    if sinstance(value, string_types):
```
Comment (elemoine) [inline c2cgeoportal/scaffolds/__init__.py]: `isinstance` :)

Comment (sbrunner) [inline c2cgeoportal/scaffolds/__init__.py]: fixed :-)

**Comment by elemoine** [comment]
Please merge.

**Comment by pgiraud** [comment]
I checked what the `six` package is. It seems to be related to python 2 & 3 compatibility.
Is c2cgeoportal supposed to be run with python3?
If not, the following test would be more relevant:
```python
if isinstance(value, basestring):
```

**Comment by sbrunner** [comment]
Actually no but we should prepare the migration

**Pull request merged**